### ISSUE TITLE
[net-config] Please optimize all API docs

### ISSUE THREAD

**Issue by cuiyanx** [01org/zephyr.js#1403]
### Description

In aio:
```
AIOPin.on
void on(string eventType, ReadCallback callback);
```
In ble:
```
BLE.on
void on(string eventType, EventCallback callback);
```
In gpio:
```
GPIOPin.onchange
attribute ChangeCallback onchange;
```
In net-config:
```
NetConfig.onup Event
void onup(void);

NetConfig.ondown Event
void ondown(void);
```
What is rule for event? If I only read api docs, I will can not use it. 
So, please optimize all API docs. 

### Test Builds

Branch | Commit Id | Target Device | Test Date | Result
---|---|---|---|---
master | 5726f4e | Arduino 101 | July 25, 2017 | Fail

**Comment by grgustaf** [comment]
Are you pointing out the inconsistency between using on() in one API but onchange in another? Yes, we should make them all use a common method.

Or are you really asking for something to change in the documentation alone? If so, I'm confused about what you're asking for.

**Comment by grgustaf** [comment]
OK, @jprestwo helped me understand that you mean events are documented in different ways. I think we would probably be wise to move our documentation to be formatted more like Node.js documentation instead of WebIDL-like. So that has a specific way of handling Event documentation. Mission accepted. :)

**Comment by grgustaf** [comment]
Fixed in #1470.

**Comment by cuiyanx** [comment]
Fixed and closed
